src/solidOpts.ts

```diff
@@ -21,7 +24,24 @@ let elementDeleteQueue: ElementNode[] = [];
 function flushDeleteQueue(): void {
   for (let el of elementDeleteQueue) {
     if (Number(el._queueDelete) < 0) {
-      el.destroy();
+      if (el.preserve) {
+        el.alpha = 0;
+        if (!el._hasCleanup) {
+          el._hasCleanup = true;
+          const removeIt = el.destroy.bind(el);
+          s.runWithOwner(el.preserve, () => {
+            s.onCleanup(() => {
+              removeIt();
```

Comment:
Why not just `el.destroy()` here?

Reply:
Was testing when cleanup was occurring - this is a WIP